### ISSUE TITLE
PyAV version/fork question

### ISSUE THREAD

**Issue by bpinsard** [pupil-labs/pupil#2067]
Hi,
It is more a question than an issue, I was wondering why PyAV version was fixed to a forked past version?  I cannot find anywhere that information.
I implemented in pupil the use of libx265 for storing grayscale videos for performance/better compression results with our camera.
However, I am having displaying problem in the player, while changes in newer PyAV version allow better handling of grayscale video.

Are there any plan to update that dependency? Is it a difficult change ?

Thanks.

**Comment by papr** [comment]
@bpinsard We forked the repository a few years ago because we needed our pyav version to includes specific changes that could not have been merged into upstream at that time. We are thinking about extracting these changes into a separate cython library, but we are waiting for https://github.com/PyAV-Org/PyAV/issues/716 to be resolved before we can proceed with that.

**Comment by bpinsard** [comment]
Thanks for the explanation! Closing now.
